src/desdemona/lifecycles/metrics.clj

```diff
@@ -0,0 +1,13 @@
+(ns desdemona.lifecycles.metrics)
+
+(defn add-metrics
+  "Add's throughput and latency metrics to a task"
+  [job task opts]
+  (update-in
+   job
+   [:lifecycles]
```

Comment:
Style nitpick: `(update-in job [a] ...)` => `(update job a ...)`

Not sure why kibit doesn't pick that up. Sounds like a feature :)